### DISCUSSION TITLE
How do I add my own Sqlite DB?

### DISCUSSION THREAD

**Discussion by JenuelDev** [tekartik/sqflite#965] [Q&A] [answered]
So, I have created a db called `dictionary.db`, I am modifying it for more definitions. And I want to use it.. How do I add it in my project? Thanks for a lot of help.

**Comment by alextekartik** [comment, marked as answer]
One solution is to add the database as an asset. More information on how to open a database from an asset explained here: https://github.com/tekartik/sqflite/blob/master/sqflite/doc/opening_asset_db.md
